sql/step_100_add_delete_btn_in_medicalstock.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `oh_menuitem` VALUES ('btnpharmstockcmovdelete','angal.common.delete.btn','angal.common.delete.btn','x','X','medicalstock','none','N',3);
```

Comment:
Please see the naming convention used in another PR:   https://github.com/informatici/openhospital-core/pull/1241/files#diff-30fbda93a84498a0e72d8670e216445f20349198a65e4cac507c97e2336cf4c2  and https://github.com/informatici/openhospital-core/pull/1241/files#diff-478eae7600e2ef8b726e86922e614a877a4f9d8a4be8e81d39a8990d3959953b

Reply:
Don't you also need to add something to `load_demo_data.sql`

Reply:
okay I added also now